modules/API/external/external_api.js

```diff
@@ -319,16 +319,9 @@ class JitsiMeetExternalAPI extends EventEmitter {
      * @returns {void}
      */
     dispose() {
-        const frame = document.getElementById(this.frameName);
-
         this.postis.destroy();
-        if (frame) {
-            frame.src = 'about:blank';
-        }
-        window.setTimeout(() => {
-            this.iframeHolder.removeChild(this.frame);
-            this.iframeHolder.parentNode.removeChild(this.iframeHolder);
-        }, 10);
+        this.removeAllListeners();
```

Comment:
Since you are refactoring this, can you add some protection for double-disposal?